ursula_cli/shell.py

```diff
@@ -427,6 +430,9 @@ def main():
     parser.add_argument('--provisioner',
                         help='The external provisioner to use',
                         default=None, choices=["vagrant", "heat"])
+    parser.add_argument('--heat-stack-name',
+                        help='Name of the heat stack when heat provisioner is used',
+                        default=None)
```

Comment:
The default value of this arg will be set in another code change block which is located in funtion of _run_heat(), which means that we don't need a default value for it unless the heat provisioner is enabled. : )